test/e2e/network/service.go

```diff
@@ -1978,13 +1978,6 @@ var _ = SIGDescribe("Services", func() {
 					},
 				},
 			},
-			Lifecycle: &v1.Lifecycle{
```

Comment:
I think the deletion of this is causing the test to fail

Reply:
if it's using the proper agnhost image and command then this shoudn't be needed, I don't think?

Reply:
That's what was [considered](https://github.com/kubernetes/kubernetes/pull/79122#issuecomment-518856820), I'll try and implement this